docs/merged_objects.md

```diff
@@ -132,44 +132,45 @@ metadata(merged_sce)$library_metadata$SCPCL000000
+| `af_num_cells`                 | Number of cells reported by `alevin-fry`                                                                                                                                                                                                                                                                                                                                           |
+| `tech_version`                 | A string indicating the technology and version used for the single-cell library, such as 10Xv2, 10Xv3, or 10Xv3.1                                                                                                                                                                                                                                                                  |
+| `assay_ontology_term_id`       | A string indicating the [Experimental Factor Ontology](https://www.ebi.ac.uk/ols/ontologies/efo) term id associated with the `tech_version`                                                                                                                                                                                                                                        |
+| `seq_unit`                     | `cell` for single-cell samples or `nucleus` for single-nucleus samples                                                                                                                                                                                                                                                                                                             |
+| `transcript_type`              | Transcripts included in gene counts: `spliced` for single-cell samples and `unspliced` for single-nuclei                                                                                                                                                                                                                                                                           |
+| `sample_metadata`              | Data frame containing metadata for each sample included in the library (see the [`Sample metadata` section below](#singlecellexperiment-sample-metadata))                                                                                                                                                                                                                          |
+| `sample_type`                  | A string indicating the type of sample, with one of the following values: `"patient-derived xenograft"`, `"cell line"`, or `"patient tissue"`. If the library is multiplexed, this will be a named vector giving the sample type for each sample ID in the object                                                                                                                  |
```

Comment:
```suggestion
| `sample_type`                  | A string indicating the type of sample, with one of the following values: `"patient-derived xenograft"`, `"cell line"`, or `"patient tissue"`. If the library is multiplexed, this will be a named vector giving the sample type for each sample ID in the library                                                                                                                  |
```
I think we want to say library here, because all of this information will be in a nested list by library ID in the merged object.